ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -101,7 +101,9 @@
       "comments": [
         "After we add support for binary FK-joins, this test should still fail, but with a different error:",
         "The new error should point out that FK-joins cannot be used in n-way joins yet.",
-        "Note: this case should be supported after we lift the general restriction to not support FK-joins in n-way joins."
+        "Note: this case should be supported after we lift the general restriction to not support FK-joins in n-way joins.",
+        "TODO: this test should be deleted once the feature is ungated and the feature flag is removed",
```

Comment:
Should we no remove the other test after removing the feature flag, as we don't need to enable the feature in tests any longer?

Reply:
No, we'll need to remove this one because the error message is different and the other copy of the test has the correct error message.